Hymns/Display/HymnLyricsView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import Resolver
+
+public struct HymnLyricsView: View {
+    
+    @ObservedObject private var viewModel: HymnLyricsViewModel
+    
+    init(viewModel: HymnLyricsViewModel) {
```

Comment:
What is the benefit of using init(.....)     instead of just having viewModel: HymnLyricsViewModel?

Reply:
I'm not quite sure what you are asking.

If you are asking why we can't just do private var viewModel: HymnLyricsViewModel = HymnLyricsViewMode(), that's because we are injecting the view model via dependency injection

If you are asking why we can't just do @injected var viewModel: HymnLyricsViewModel, it's because we need to create the HymnLyricsView from the container view, so (afaik, and I might be wrong), we can use the magical @injected incantation because we are manually creating that view, and thus have to pass the dependencies through the constructor